src/main/java/com/prgms/VoucherApp/domain/customer/storage/BlackListFileStorage.java

```diff
@@ -0,0 +1,56 @@
+package com.prgms.VoucherApp.domain.customer.storage;
+
+import com.prgms.VoucherApp.domain.customer.Customer;
+import com.prgms.VoucherApp.domain.customer.CustomerStatus;
+import com.prgms.VoucherApp.domain.customer.dto.CustomerDto;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.stereotype.Repository;
+
+import javax.annotation.PostConstruct;
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Paths;
+import java.util.List;
+import java.util.Map;
+import java.util.UUID;
+import java.util.concurrent.ConcurrentHashMap;
+
+@Repository
+public class BlackListFileStorage implements BlackListStorage {
+
+    private static final Logger log = LoggerFactory.getLogger(BlackListFileStorage.class);
+    private final Map<UUID, Customer> blackListMap = new ConcurrentHashMap<>();
```

Comment:
```suggestion
    private final Map<UUID, Customer> blackListMap = new ConcurrentHashMap<>();
```
blackListMap이라는건 Map 자료구조에 의존적인 네이밍이네요. 자료구조가 변경하면 네이밍까지 변경되어야할까요..? 또 컴파일언어라서 자료구조는 다 알고 있을 거 같아서, map이라는 네임 의미가 없을 거 같아요.

Reply:
어떤 자료구조를 사용하는 중인건지 네이밍이 중요하다고 생각해서 map이라는 네이밍을 사용하였는데 자료구조에 의존적인 네이밍은 안 좋은걸까요!?

Reply:
네 만약 자료구조를 변경하면 네이밍도 변경되어야하니까요..!
또 타입 언어라서 Map이라는게 명시되어야하는데, 굳이 쓸 필욘 없어보여요.